c2corg_ui/templates/route/edit.html

```diff
@@ -297,11 +297,11 @@ <h2 class="heading show-phone"><span translate>numbers</span></h2>
             ## ROUTE LENGTH
             <div id="route_length-group" class="data half form-group" ng-class="{ 'has-error': editForm.route_length.$touched && editForm.route_length.$invalid,'has-success': route.route_length }"
                  ng-if="route.activities.indexOf('skitouring') > -1 || route.activities.indexOf('snow_ice_mixed') > -1 || route.activities.indexOf('hiking') > -1 || route.activities.indexOf('snowshoeing') > -1
-                           || route.activities.indexOf('via_ferrata') > -1 || route.activities.indexOf('mountain_biking') > -1 || route.activities.indexOf('rock_climbing') > -1">
```

Comment:
I don't remember precisely because I made the change 2 weeks ago in the initial version of the PR. But I think I double checked in the google spreadsheet, yes.